.nvmrc

```diff
@@ -0,0 +1 @@
+6.10.0
```

Comment:
By default we're doing several `npm install` commands, which are installing Node packages that require a later version than Node 0.10.

This means that if I don't provide a higher default Node version, by default this build will fail. It will only work if users manually override the Node version to be high enough.

I think it would make more sense to use a high enough Node version by default. Do you think overriding `DEFAULT_VERSION` would be a better way to do that?